lib/main_common.pm

```diff
@@ -1529,7 +1529,7 @@ sub load_extra_tests_console {
 }
 
 sub load_extra_tests_docker {
-    return unless check_var('ARCH', 'x86_64');
+    return unless get_var('ARCH', '') =~ /x86_64/ppc64le/aarch64;
```

Comment:
sorry, that's not a valid regex, it needs to be `|` as separation. However, I prepared https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6598 as a replacement as we need to exclude the tests on SLE anyway.